src/Symfony/Component/Config/Resource/GlobResource.php

```diff
@@ -91,7 +91,7 @@ public function __sleep(): array
         return ['prefix', 'pattern', 'recursive', 'hash', 'forExclusion', 'excludedPrefixes'];
     }
 
-    public function getIterator()
+    public function getIterator(): iterable
```

Comment:
getIterator must return \Traversable as I said several times now. an array (iterable) would not be valid.

Reply:
https://3v4l.org/9mPes